config/application.rb

```diff
@@ -29,5 +29,15 @@ class Application < Rails::Application
       generate.controller_specs false
       generate.system_tests false
     end
+
+    config_files = ['secret.yml']
+
+    config_files.each do |file_name|
+      file_path = File.join(Rails.root, 'config', file_name)
+      config_keys = HashWithIndifferentAccess.new(YAML::load(IO.read(file_path)))[Rails.env]
+      config_keys.each do |k,v|
+        ENV[k.upcase] ||= v
+      end
+    end
```

Comment:
Please explain what this code is doing for you.

Reply:
Nothing! Part of the solution to remove it